modules/monitoring-configuring-alert-receivers.adoc

```diff
@@ -60,11 +60,11 @@ Alternatively, you can navigate to the same page through the notification drawer
 +
 .. Select *Show advanced configuration* if you want to edit the default option not to send resolved alerts to the receiver or edit the icon and username configuration. You can also choose whether to find and link channel names and usernames.
 
-. By default, firing alerts with labels that match all of the selectors will be sent to the receiver. If you want label values for firing alerts to be matched exactly before they are sent to the receiver:
-.. Add routing label names and values in the *Routing Labels* section of the form.
-+
-.. Select *Regular Expression* if want to use a regular expression.
-+
-.. Select *Add Label* to add further routing labels.
+. By default, firing alerts with labels that match all of the selectors are sent to the receiver. If you want label values for firing alerts to be matched exactly before they are sent to the receiver, perform the following steps:
+.. Add routing label names and values in the *Routing labels* section of the form.
+
+.. Select *Regular expression* if want to use a regular expression.
```

Comment:
4.12/4.13, there is `Regular expression` on page, see picture
![4 13](https://github.com/juzhao/test/assets/22743651/f369f4fa-198e-48eb-bf9a-0081eda92007)
since 4.14, there is not `Regular expression`on page, see picture
![4 14](https://github.com/juzhao/test/assets/22743651/a9f8f7b6-e109-4405-bf9f-905599074109)

Reply:
Thank you for pointing this out! I will solve this by removing this line and then I will manually cherrypick to older versions to include the line. Thank you!